presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -42,7 +42,7 @@
 import com.facebook.presto.spi.connector.ConnectorMetadata;
```

Comment:
nit: this commit message should not mention Hive, it's about making SPI privilege consistent with the SQL standard